helm/hello-world-app/README.md

```diff
@@ -0,0 +1,34 @@
+# hello-world-app
+
+![Version: 0.2.1](https://img.shields.io/badge/Version-0.2.1-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+A chart that deploys a basic hello world site and lets you test values merging of user values configmap and secrets.
+
+**Homepage:** <https://github.com/giantswarm/hello-world-app>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Giant Swarm - Honeybadger |  |  |
```

Comment:
This doesn't look great. Should we somehow add more values in the table or remove columns?

Reply:
Yeah, we can add more chart fields

Reply:
I think it doesn't hurt. It's generated based on the default template, and I wouldn't require adapting that one, as there isn't much to gain.